testcharms/charm-repo/quantal/lxd-profile-fail/lxd-profile.yaml

```diff
@@ -5,6 +5,7 @@ config:
   linux.kernel_modules: openvswitch,nbd,ip_tables,ip6_tables
   # boot.* keys are blacklisted by juju.
   boot.autostart: "true"
+  limits.memory: 256mb
```

Comment:
Causes profile to fail even with force:
cannot write lxd profiles: Invalid value: 256mb